ui/nuxeo-routing-behavior.js

```diff
@@ -161,7 +172,7 @@ export const RoutingBehavior = {
     if (!routeVal) {
       throw new Error(`invalid router key: ${routeKey}`);
     }
-    return fn(routeVal, ...params);
+    return { path: fn(routeVal, ...params), repo };
```

Comment:
we could return a ready to go multirepo path here, no ?

Reply:
Not without refactoring something else, because this is used by both `urlFor` and `navigateTo`, but the former requires the full url, with the hashbang, whereas the latter requires only the url after the #! (because it uses page to navigate, which expect the path after the #!). This means that this method needs to provide them both separately so the correct path can be put together correctly for both cases. This would be simplified if we dropped the usage of `page(...)` for navigation I believe, but I'm not sure if it would not have other implications.